docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -159,7 +159,7 @@ If there is an unreleased feature you would like to try, it may be available in
   <dependency>
     <groupId>software.amazon.jdbc</groupId>
     <artifactId>aws-advanced-jdbc-wrapper</artifactId>
-    <version>2.2.6-SNAPSHOT</version>
+    <version>2.3.0-SNAPSHOT</version>
```

Comment:
The snapshot version should be one minor version further, so 2.3.1 in this case I think